code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -107,15 +107,21 @@
 	if(cooldowntime > world.time)
 		to_chat(user, "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>")
 		return
-	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Nar-Sien Hardsuit")
+	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Bastard Sword")
 	var/pickedtype
 	switch(choice)
 		if("Shielded Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/cult_shield
 		if("Flagellant's Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/berserker
-		if("Nar-Sien Hardsuit")
-			pickedtype = /obj/item/clothing/suit/space/hardsuit/cult
+		if("Bastard Sword")
+			if(world.time >= 13200)
```

Comment:
Put this one behind some define. Also it should count from round start.

Reply:
Are you saying that world.time counts from roundstart or that I need to use a different timer?

Reply:
The latter, world.time counts from server startup.

Reply:
Yea but what timer works from roundstart? World.time seems to be the standard and it includes built-in assumptions about how long it takes the round to start.

Reply:
Ticker stores roundstart time, you just need to subtract that.